lib/qunit-bdd.js

```diff
@@ -270,7 +271,14 @@
     }
     context.tests.push({
       description: description,
-      body: body
+      body: function() {
+        try {
+          body();
+        } catch (e) {
+          e.stack && console.warn(e.stack);
```

Comment:
Why is this stack not being reported by QUnit? Or is it, and it's simply not being logged so Chrome won't make it into a link?